nav2_behavior_tree/include/nav2_behavior_tree/bt_action_node.hpp

```diff
@@ -152,10 +161,10 @@ class BtActionNode : public BT::CoroActionNode
         return BT::NodeStatus::SUCCESS;
```

Comment:
I see you made the `on_aborted` and `on_cancelled` return a code. Can you update the `on_succeeded` to do the same for consistency?

Reply:
My reasoning is that it is very hard to imagine someone that wants to return FAILURE when the action succeeded, but on the other hand, it is good for clean ups...

i will